raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,332 @@
+            - assert: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+      - serial:
+          name: "2 tries different cases"
+          tasks:
+            - open_channel: {from: 2, to: 4, total_deposit: 0, expected_http_status: 201}
+            - wait: 10
+            - deposit: {from: 2, to: 4, total_deposit: 100_000_000_000_000_000}
+            - wait: 10
+            # Try to make a deposit smaller the the amount that was deposited
+            - deposit: {from: 2, to: 4, total_deposit: 30_000_000_000_000_000, expected_http_status: 409}
+            # Make a deposit bigger than the deposited amount
+            - deposit: {from: 2, to: 4, total_deposit: 150_000_000_000_000_000}
+            - wait: 10
+      - parallel:
+          name: "Verify the channels"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+            # New channel
+            - assert: {from: 2, to: 4, total_deposit: 150_000_000_000_000_000, balance: 150_000_000_000_000_000, state: "opened"}
+            - assert: {from: 4, to: 2, total_deposit: 0, balance: 0, state: "opened"}
+      - serial:
+          name: "Continue building the network"
+          tasks:
+            # 0 opens a channel with 4
+            - open_channel: {from: 0, to: 4, total_deposit: 10_000_000_000_000_000, expected_http_status: 201}
+            - assert: {from: 0, to: 4, total_deposit: 10_000_000_000_000_000, balance: 10_000_000_000_000_000, state: "opened"}
+            # 4 deposits 25 tokens onto the 0 - 4 channel
+            - deposit: {from: 4, to: 0, total_deposit: 25_000_000_000_000_000}
+            - wait: 20
+            # 0 performs a payment to 4 of 50 tokens (path 0<->1<->2<->4)
```

Comment:
Again here. Please update the value. 

And can you please in general write `node0` or `nodeX` instead of just 0 or 1? This makes it easier to read and it complies with how it's done in all other scenarios.